services/crud.py

```diff
@@ -95,5 +95,11 @@ def remove_user(db: Session, user_id: int):
         raise HTTPException(status_code=500, detail="An error occurred while deleting user")
```

Comment:
we don't raise http exceptions in the service layer , create an exception folder for costume exceptions then raise that exception and catch it in the api layer where you raise http exception